<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>601</x>
      <y>31</y>
      <width>289</width>
      <height>85</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>ESFERA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="1">
       <widget class="QLCDNumber" name="lcdNumber_5"/>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QSlider" name="horizontalSlider_REsfera">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_RaioEsfera">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Raio</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>601</x>
      <y>122</y>
      <width>289</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>ELIPSE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0" rowspan="3">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_RxElipse">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Rx</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="horizontalSlider_RxElipse">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_RyElipse">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ry</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="horizontalSlider_RyElipse">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_RzElipse">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Rz</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="horizontalSlider_RzElipse">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QLCDNumber" name="lcdNumber"/>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="lcdNumber_2"/>
      </item>
      <item row="2" column="1">
       <widget class="QLCDNumber" name="lcdNumber_4"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>601</x>
      <y>299</y>
      <width>289</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>CAIXA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0" rowspan="3">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_XCaixa">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>DimX</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="horizontalSlider_DimXCaixa">
          <property name="pageStep">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_YCaixa">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>DimY</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="horizontalSlider_DimYCaixa">
          <property name="pageStep">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_ZCaixa">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>DimZ</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="horizontalSlider_DimZCaixa">
          <property name="pageStep">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QLCDNumber" name="lcdNumber_6"/>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="lcdNumber_7"/>
      </item>
      <item row="2" column="1">
       <widget class="QLCDNumber" name="lcdNumber_8"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>601</x>
      <y>476</y>
      <width>289</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>COR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0" rowspan="3">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_RCor">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Red</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="horizontalSlider_RCor">
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_GCor">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Green</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="horizontalSlider_GCor">
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_BCor">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Blue</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="horizontalSlider_BCor">
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QLCDNumber" name="lcdNumber_9"/>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="lcdNumber_10"/>
      </item>
      <item row="2" column="1">
       <widget class="QLCDNumber" name="lcdNumber_11"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>481</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>PLANO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <widget class="QLabel" name="label_ZPlano">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="horizontalSlider_ZPlano">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLCDNumber" name="lcdNumber_3"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="horizontalSlider_PlanoY">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLCDNumber" name="lcdX"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="horizontalSlider_PlanoX">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLCDNumber" name="lcdY"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>421</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_12">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Começar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_Esfera">
       <property name="text">
        <string>Esfera</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_TirarEsfera">
       <property name="text">
        <string>TirarEsfera</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_Elipse">
       <property name="text">
        <string>Elipse</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_TirarElipse">
       <property name="text">
        <string>TirarElipse</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_Caixa">
       <property name="text">
        <string>Caixa</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_TirarCaixa">
       <property name="text">
        <string>TirarCaixa</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_Voxel">
       <property name="text">
        <string>Voxel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_TirarVoxel">
       <property name="text">
        <string>TirarVoxel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionSalvar"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_ZPlano</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_RyElipse</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>815</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_DimXCaixa</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_6</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>819</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_DimZCaixa</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_8</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>734</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>814</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_REsfera</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>711</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>830</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_RxElipse</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>774</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>825</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_DimYCaixa</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_7</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>809</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_RzElipse</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_4</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>747</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>826</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>mataTudo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSalvar</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>Salvar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_RCor</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_9</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>767</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>809</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_GCor</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_10</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>753</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>810</x>
     <y>587</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_BCor</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_11</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>749</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>811</x>
     <y>618</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_PlanoX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdY</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_PlanoY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdX</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>mataTudo()</slot>
  <slot>Salvar()</slot>
  <slot>Abrir()</slot>
 </slots>
</ui>
